### PR TITLE
cmake: enable write-back cache in spec

### PR DESCRIPTION
**Pull request by lixiaoy1**
Signed-off-by: Li, Xiaoyan <xiaoyan.li@intel.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by lixiaoy1** [comment]
@dillaman Please help to review.

File: ceph.spec.in [hunk 1]
```diff
@@ -62,8 +62,6 @@
 %endif
 %bcond_with seastar
 %bcond_with jaeger
-%bcond_with rbd_rwl_cache
```
Comment (dillaman) [inline ceph.spec.in:65]: Just switch these to `bcond_without`

Comment (lixiaoy1) [reply ceph.spec.in:65]: Done. Thanks.

**Review by dillaman — APPROVED** [review]
lgtm -- thx

**Comment by smithfarm** [comment]
@lixiaoy1 @dillaman Are you planning to backport this further than to pacific? I'm asking because it appears to have broken the ceph build in OBS. I have a fix up for review at https://github.com/ceph/ceph/pull/39755. Thanks!

**Comment by dillaman** [comment]
> @lixiaoy1 @dillaman Are you planning to backport this further than to pacific? I'm asking because it appears to have broken the ceph build in OBS. I have a fix up for review at #39755. Thanks!

Negative.

**Pull request merged**